fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneQueryClause.java

```diff
@@ -146,6 +147,10 @@ protected static Map<String, Set<String>> getHighlightingTermsMap(@Nonnull final
             BitSetQuery bitsetQuery = (BitSetQuery)query;
             String field = bitsetQuery.getField();
             highlightingTermsMap.computeIfAbsent(field, key -> new HashSet<>()).add(field.toLowerCase(Locale.ROOT));
+        } else if (query instanceof PointRangeQuery) {
```

Comment:
What is the intent here? IIUC, this change would allow queries with a range of int(long) point to be included. Is the idea to allow them to be matching the doc but then ignored for highlight? Is the idea to do the same for term range ("a" TO "b")?

Reply:
Yes, we want to allow matching using a value or a range. If there's no text part in the clause, then the highlighting would be ignored (say if `isExpunged: 0` is the only predicate) and `highlights` would be empty in the response.

We wouldn't send a`group:  ["a" TO "b"]` predicate. But `group: "a"` would be a TermQuery and that's already allowed. Either way, tha `"a"` and `"b"` should be ignored for highlighting